keyboards/gonnerd/keymaps/gam3cat/keymap.c

```diff
@@ -11,10 +11,15 @@ enum layers {
     _AL,        // Adjust Layer
 };
 
-enum gonnerd_keycodes {
+enum custom_keycodes {
     DYNAMIC_MACRO_RANGE = SAFE_RANGE,
+    QMK_REV,
+    KC_WEB,
+    KC_SP4
 };
 
+static uint8_t current_layer;
+
 #include "dynamic_macro.h"
 #define _______ KC_TRNS
```

Comment:
These defines aren't necessary, as these are actually defined in the core.  You can safely remove them. 

https://github.com/qmk/qmk_firmware/blob/a0309db983150fd0197eb620a6ba552d90d29f93/quantum/quantum_keycodes.h#L30-L31


```suggestion
```